.babelrc

```diff
@@ -43,6 +45,7 @@
         "react"
     ],
     "plugins": [
+        "babel-plugin-polished",
```

Comment:
These three occurrences prevent the project from compiling since the plugin doesn't exist anymroe